lib/stm32/h7/pwr.c

```diff
@@ -0,0 +1,50 @@
+/** @defgroup pwr_file PWR peripheral API
+ *
+ * @ingroup peripheral_apis
+ *
+ * @brief <b>libopencm3 STM32H7xx Power Control</b>
+ *
+ * @version 1.0.0
+ *
+ * @author @htmlonly &copy; @endhtmlonly 2011 Stephen Caudle <scaudle@doceme.com>
+ * @author @htmlonly &copy; @endhtmlonly 2017 Matthew Lai <m@matthewlai.ca>
+ *
+ * @date 12 March 2017
+ *
+ * This library supports the power control system for the
+ * STM32H7 series of ARM Cortex Microcontrollers by ST Microelectronics.
+ *
+ * LGPL License Terms @ref lgpl_license
+ */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2011 Stephen Caudle <scaudle@doceme.com>
```

Comment:
I'm convinced this contains any of the caudle code, which is only the version in the f4 file, and the caudle copyright was carried over to the f7 file, even though it too has nothing left other than a function name.  anyway. yucky :)